dockerfiles/init/manifests/codenvy.pp

```diff
@@ -259,6 +259,7 @@
   $machine_ws_agent_run_command = getValue("CODENVY_MACHINE_WS_AGENT_RUN_COMMAND","~/che/ws-agent/bin/catalina.sh run")
 # Docker privilege mode, default false
   $machine_docker_privileged = getValue("CODENVY_MACHINE_DOCKER_PRIVILEGED","false")
+  $machine_docker_securityopt = getValue("CODENVY_MACHINE_DOCKER_SECURITYOPT","")
```

Comment:
as in env file `CODENVY_MACHINE_DOCKER_SECURITYOPT` is commented default value will be `""` is that expected?

Reply:
AFAIK yes
https://github.com/eclipse/che/pull/6856/files

in che, default value is 
```
che.docker.securityopt=
```